docs/features.md

```diff
@@ -48,16 +48,19 @@ If a `pulse` is not specified, then a disruption will not be pulsing.
 
 ## Targeting
 
+**NEW:** StaticTargeting currently defaults to true. It will default to false after some transition time. Read StaticTargeting.
+
```

Comment:
You can link directly to the StaticTargeting header within the doc here, That would be nice to turn "Read StaticTargeting" into a hyperlink

Reply:
Done, TIL